src/main/ui/app/component/AddOnList.jsx

```diff
@@ -1,21 +1,43 @@
 import {Component} from "react";
 import AddOn from "./AddOn";
+import { ListGroup , Table} from 'react-bootstrap';
+
 
 export default class AddOnList extends Component {
 
     render() {
         if (!this.props.addons) {
-            return <div>Loading...</div>
+            return (
+             <section>
+      <p class="lead textdec">Loading</p>
+      </section>
+      )
         }
         else if (this.props.addons.length === 0) {
-            return <div>No results</div>
+            return (
+            <section>
+      <h1 class="lead textdec">No Results</h1>
+      </section>
+
+            )
         }
         else {
             return (
-                    <div>
+                   <div>
+		   <Table className="maintable">
+    			<thead>
```

Comment:
Looking at the actual UI, I don't see a need for this header. It's pretty obvious what things are, and this takes up valuable space toward the top of the screen.